app/views/catalog/_home_text.html.erb

```diff
@@ -13,26 +13,36 @@ Unless required by applicable law or agreed to in writing, software distributed
   specific language governing permissions and limitations under the License.
 ---  END LICENSE_HEADER BLOCK  ---
 %>
-<h2 id="featured_content_header">Sample Content</h2>
+<h2 id="featured_content_header">Featured Content</h2>
 <div id="featured_content" class="row">
   <div class="col-md-4">
-    <h4>Featured Collection</h4>
-    <a href="#">
-      <%= image_tag "Nutcracker-Ballet.png", alt: "Featured Collection" %>
+    <h4>Featured Video Collection</h4>
+    <a href="/catalog?f%5Bcollection_ssim%5D%5B%5D=Nosotros+Television+Programme&q=&search_field=all_fields" class="embed-responsive embed-responsive-16by9">
+      <%= video_tag "welcome0.mp4", autoplay: 'true', alt: "Featured Video Collection", class: "embed-responsive-item"%>
```

Comment:
Maybe name the videos to something more informative then `welcome0` to `welcome3`?